operator/pkg/helmreconciler/apply.go

```diff
@@ -211,4 +226,16 @@ func (h *HelmReconciler) ApplyObject(obj *unstructured.Unstructured) error {
 		}
 		return nil
 	})
+
+}
+
+// use server-side apply, require kubernetes 1.16+
+func (h *HelmReconciler) serverSideApply(obj *unstructured.Unstructured) error {
+	scope.Info("using server side apply to update obj")
```

Comment:
put the ojbect name in the log? to match the old logic

Reply:
done